scripts/emailings/mailing-send.php

```diff
@@ -268,6 +268,13 @@
 
 						$substitutionisok = true;
 
+						$moreinheader = '';
+						if (preg_match('/__UNSUBSCRIBE__/', $message)) {
+							$moreinheader = make_substitutions('__UNSUBSCRIBE__', $substitutionarray);
```

Comment:
Why not just, in 2 lines
```
$moreinheader = "List-Unsubscribe: <__UNSUBSCRIBE__>\n";
$moreinheader = make_substitutions($moreinheader, $substitutionarray);
```

Reply:
Yes,I did check and the link correctly shown